frontend/src/App.vue

```diff
@@ -19,4 +25,9 @@ const userStore = useUserStore();
 .app {
   display: flex;
 }
+
+.rsprd-main {
+  display: block;
+  width: 100%;
```

Comment:
Nitpick: elements with a `display` set to `block` should already try to expand to full width. There should be no need to specify it explicitly.

Reply:
That is true, but display is set to flex on its parent. So it doesn't stretch to fit the whole width, it is as wide as its contents. But I changed width to flex property because it seems more appropriate and set it to fit the remaining width.